formats/specialist_document/publisher/details.json

```diff
@@ -880,6 +881,64 @@
         }
       }
     },
+    "utaac_decision_metadata": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "bulk_published": {
+          "type": "boolean"
+        },
+        "document_type": {
+          "type": "string",
+          "enum": [
+            "utaac_decision"
+          ]
+        },
+        "tribunal_decision_judges": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_judges_name": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_categories": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_categories_name": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_sub_categories": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_sub_categories_name": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "tribunal_decision_decision_date": {
+          "type": "string",
+          "pattern": "^[1-9][0-9]{3}-(0[1-9]|1[0-2])-([012][0-9]|3[0-1])$"
```

Comment:
We followed this repo's instructions for [converting finder schemas](https://github.com/alphagov/govuk-content-schemas/blob/master/docs/converting-finder-schemas.md#converting-finder-schemas).

So the [FinderSchemaConverter](https://github.com/alphagov/govuk-content-schemas/blob/master/lib/govuk_content_schemas/finder_schema_converter.rb) was run on the [UTAAC schema](https://github.com/ministryofjustice/specialist-publisher/blob/master/finders/schemas/administrative-appeals-tribunal-decisions.json).

The [converter outputs that regexp pattern](https://github.com/alphagov/govuk-content-schemas/blob/master/lib/govuk_content_schemas/finder_schema_converter.rb#L147) in place of [date fields from the schema](https://github.com/ministryofjustice/specialist-publisher/blob/master/finders/schemas/administrative-appeals-tribunal-decisions.json#L2379).

Do GDS want to change the converter to output `"type": "date"` instead? If yes, then all other dates in the `details.json` probably should be changed to `"type": date` then. Maybe that could be done in a separate pull request, changing the UTAAC entry at that time?